linux/src/async_socket_linux.c

```diff
@@ -205,7 +205,7 @@ static int thread_worker_func(void* parameter)
                     {
                         void* send_pos = send_context->buffers[index].buffer + send_size;
                         uint32_t send_length = send_context->buffers[index].length - send_size;
-                        send_size += send(send_context->socket_handle, send_pos, send_length, 0);
+                        send_size += send(send_context->socket_handle, send_pos, send_length, MSG_NOSIGNAL);
```

Comment:
>MSG_NOSIGNAL [](http://example.com/codeflow?start=94&length=12)

I would love a comment about why we need this. No-one except The Jelani will know in a couple of weeks.

Reply:
This would be a good thing for a spec, but all in due time.